### PR TITLE
Fix timeout when deleting many submissions at a time.

### PR DESCRIPTION
**Pull request by noliveleger**
## Description

Improve (drastically) the time to delete multiple submissions at a time.

## Additional infos 

Depending on the load of the server, deleting few hundred of submissions should take few seconds now although it could have taken more than 2 minutes before (and time out).

## Related issues 

Blocked by #848

### CODE REVIEW

**Comment by noliveleger** [comment]
@joshuaberetta, if you're happy with the PR it, please do not merge it until #848 is reviewed/merged. You can "accept" it though :-D

**Review by joshuaberetta — APPROVED** [review]
Impressive performance boost! 🎉 
Deleting 2500+ submissions:
- before: 47.5s
- after: 1.2s

Deleting 15k+ submissions:
- before: 60s
- after: 8s

File: onadata/apps/logger/models/instance.py
Comment (noliveleger) [reply onadata/apps/logger/models/instance.py:209]: I'd rather keep the way it is and avoid importing it each time we enter the function.

File: onadata/apps/logger/models/instance.py
Comment (jnm) [inline onadata/apps/logger/models/instance.py:181]: Same question as above about removing the `sender` parameter

**Review by jnm — CHANGES_REQUESTED** [review]

File: onadata/apps/logger/models/instance.py
Comment (jnm) [reply onadata/apps/logger/models/instance.py:102]: Yes, let's revert it per our discussion because Django says the receivers always need to accept `sender`, and there's no guarantee that `sender` will always be passed as a keyword argument instead of positionally

**Review by jnm — APPROVED** [review]

**Pull request merged**
